### ISSUE TITLE
Not able to Run Spring Boot 1.3.0M5

### ISSUE THREAD

**Issue by jforjava1981** [spring-projects/spring-boot#3961]
I tried changing Spring Boot version from 1.2.2 release  to 1.3.0M5. while running spring boot run I get following exception. I tried updating the snapshots while building and running but to no avail.
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.
java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAcces
sorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:483)
        at org.springframework.boot.maven.AbstractRunMojo$LaunchRunner.run(Abstr
actRunMojo.java:435)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalArgumentException: Cannot instantiate interface org.
springframework.context.ApplicationListener : org.springframework.boot.logging.C
lasspathLoggingApplicationListener
        at org.springframework.boot.SpringApplication.getSpringFactoriesInstance
s(SpringApplication.java:385)
        at org.springframework.boot.SpringApplication.getSpringFactoriesInstance
s(SpringApplication.java:362)
        at org.springframework.boot.SpringApplication.initialize(SpringApplicati
on.java:231)
        at org.springframework.boot.SpringApplication.<init>(SpringApplication.j
ava:206)
        ....
        ... 6 more
Caused by: java.lang.NoClassDefFoundError: org/springframework/context/event/Gen
ericApplicationListener
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:760)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:14
2)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:455)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:367)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at org.springframework.util.ClassUtils.forName(ClassUtils.java:250)
        at org.springframework.boot.SpringApplication.getSpringFactoriesInstance
s(SpringApplication.java:378)
        ... 10 more
Caused by: java.lang.ClassNotFoundException: org.springframework.context.event.G
enericApplicationListener
        at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 23 more

**Comment by MichaelF25** [comment]
I have a similar issue with a spring-cloud enabled application (I am using spring-cloud-starter-parent in import scope within maven). Since other spring boot projects work fine with spring boot 1.3.0.M5 I guess it's something related with spring-cloud Angel.SR3

Can you provide your maven/gradle configuration?

**Comment by php-coder** [comment]
Is it possible that you have mixed versions of Spring artifacts in your classpath? I suggest to inspect `mvn dependecy:tree` output.

**Comment by MichaelF25** [comment]
I guess the problem is caused by using spring-boot-starter-parent 1.3.0.M5 as parent and spring-cloud-starter-parent Angel.SR3 in import scope.

I'll try to create a small sample project to reproduce the issue.

**Comment by MichaelF25** [comment]
Just created a small sample via [Spring Initializr] (https://start.spring.io/) with Spring Boot V1.3.0.M5 and cloud support enabled.
The generated project uses `Brixton.M1` train version of spring-cloud-starter-parent instead of released `Angel.SR3`.
With `Angel.SR3` the above issue occurs but with `Brixton.M1` the artifact `spring-cloud-starter-security:1.1.0.M1` could not be found - but that's another issue.

To summarize: Spring Boot 1.3 is not compatible with Spring Cloud Angel.SR3 release train. This should be documented somewhere.

**Comment by snicoll** [comment]
[As described in the guidelines for contributing](https://github.com/spring-projects/spring-boot/blob/master/CONTRIBUTING.adoc#using-github-issues), we prefer to use GitHub issues only for bugs and enhancements. For general usage questions, please use [StackOverflow](http://stackoverflow.com)

**Comment by jforjava1981** [comment]
You moved this stackoverflow. Can I get a link to question? I searched on stackoverflow with spring-boot tag but couldn't find it. Or do you mean that I should move it?

**Comment by snicoll** [comment]
> For general usage questions, please use StackOverflow

Yes, please create a SO thread.

**Comment by jforjava1981** [comment]
Putting a reference link of SO question I created for any one who visits this issue in future:
http://stackoverflow.com/questions/32619700/not-able-to-run-spring-boot-1-3-0m5
